File: c++/speed.cpp
// Question asked in tcs exam
#include <iostream>
#include<bits/stdc++.h>
using namespace std;
int main() {
    float distance, time, speed_ms,speed_kmhr;
    cout<<"Enter a distance: "<<endl;
    cin>>distance;
    cout<<"Enter the time: "<<endl;
    cin>>time;
    
    speed_ms=distance/time;
    speed_kmhr=speed_ms*3.6;
    
    cout<<"Speed in m per sec: "<<speed_ms<<endl;
    cout<<"Speed in km per hr: "<<speed_kmhr<<endl;
    

    return 0;
}

File: c#/speed.cs

using System;

public class Speed {
    public static void Main(string[] args) {
       
        Console.WriteLine("Enter distance (meters):"); 
        float d = float.Parse(Console.ReadLine());

        Console.WriteLine("Enter time (seconds):");
        float t = float.Parse(Console.ReadLine());

        float div = 18.0f / 5.0f; 
        float speed = (d / t) * div;

        Console.WriteLine("Speed (km/h): " + speed);
    }
}
